plugins/VstEffect/VstEffect.cpp

```diff
@@ -76,46 +78,29 @@ VstEffect::VstEffect( Model * _parent,
 
 
 
-bool VstEffect::processAudioBuffer( SampleFrame* _buf, const fpp_t _frames )
+double VstEffect::processImpl(SampleFrame* buf, const fpp_t frames)
 {
-	if( !isEnabled() || !isRunning () )
+	assert(m_plugin != nullptr);
+	static thread_local auto tempBuf = std::array<SampleFrame, MAXIMUM_BUFFER_SIZE>();
+
+	std::memcpy(tempBuf.data(), buf, sizeof(SampleFrame) * frames);
```

Comment:
`memcpy`? You're really old-fashioned :laughing:

Reply:
It was there before so I just kept it